cypress/integration/smoke/BasicTest.ts

```diff
@@ -29,14 +29,17 @@ describe("Basic Elements", () => {
     });
   });
 
-  it("Mocked instances are visible", () => {
-    cy.ouiaType("PF4/TableRow")
-      .should("have.length", 13)
-      .eq(1)
-      .find("td")
-      .eq(0)
-      .should("have.text", "Instance one")
-      .find("a[data-testid='tableInstances-linkInstance']")
-      .should("be.visible");
+  it("Mocked instances are visible", async () => {
+    cy.route("/bridges/*").as("getBridges");
```

Comment:
@vpellegrino It seems that the `route` method might be deprecated in our cypress version (9.6.0) https://docs.cypress.io/api/commands/route. The `intercept` method has the same usage. See  https://docs.cypress.io/api/commands/intercept

Reply:
@sjamboro are you able to send edit suggestion or a pull request against my branch? Really appreciate. Thanks

Reply:
@sjamboro resolved with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/41/commits/9b0d8eeb7b5bb1fd384ecda6b4e8b1c823fa4429
Thanks for reviewing it!

Reply:
@vpellegrino I am not sure if your solution runs correctly ... see 
![image](https://user-images.githubusercontent.com/1536609/167608482-99fdfa28-2c2b-4e23-96be-6bae60e2461c.png)

There is a waiting rutine for a long time but the test is marked as succesfull.

Reply:
@vpellegrino I am sorry for delay. I was running it localy and thinking how to fix it. I have created new JIRA https://issues.redhat.com/browse/MGDOBR-710.

Reply:
@sjamboro thank you for fixing it! Please, when you'll be ready involve me in the review of MGDOBR-710. I'm willing to learn how to wait for async calls in cypress